apps/openmw/mwscript/globalscripts.cpp

```diff
@@ -125,8 +137,10 @@ namespace MWScript
         return mScripts.size();
     }
 
-    void GlobalScripts::write (ESM::ESMWriter& writer, Loading::Listener& progress) const
+    void GlobalScripts::write (ESM::ESMWriter& writer, Loading::Listener& progress)
     {
+        updateState();
```

Comment:
What is this update supposed to do in the write function? Looks wrong.

Reply:
> Also I update mRunning when writting it to the savegame, so we do not need to store the mWillBeRunning in the savegame.

Reply:
Still makes no sense to me. Why would you have to store mWillBeRunning to begin with? Its not as if you can save in the middle of script execution or even a script execution cycle.

Reply:
> Its not as if you can save in the middle of script execution or even a script execution cycle.

Then I can just update state only only once in the script execution cycle, and write() changes are indeed redundant.